.github/workflows/tests.yml

```diff
@@ -187,6 +187,13 @@ jobs:
         run: |
           ./build.sh -restore -ci -build -projects ${{ github.workspace }}/${{ matrix.project }} /bl
 
+      - name: Rebuild for Azure Functions project
```

Comment:
Can we add a comment to an issue tracking this? We will want to remove this as soon as we can.